src/Illuminate/Queue/DatabaseQueue.php

```diff
@@ -261,7 +261,7 @@ protected function getLockForPopping()
         if (Str::of($databaseVersion)->contains('MariaDB')) {
             $databaseEngine = 'mariadb';
             $databaseVersion = Str::before(Str::after($databaseVersion, '5.5.5-'), '-');
-        } elseif (Str::of($databaseVersion)->contains('vitess')) {
+        } elseif (Str::of($databaseVersion)->contains('vitess') || Str::of($databaseVersion)->contains('PlanetScale')) {
```

Comment:
You could use to simplify:
```php
Str::of($databaseVersion)->contains(['vitess', 'PlanetScale])
```

Docs: https://github.com/laravel/framework/blob/9.x/src/Illuminate/Support/Stringable.php#L170

Reply:
Nice, didn't know you could pass an array there...fixed!